ui/app/pages/create-account/connect-custodial/trustvault/email/email-screen.js

```diff
@@ -0,0 +1,187 @@
+import React, { PureComponent } from 'react'
+const PropTypes = require('prop-types')
+import Button from '../../../../../components/ui/button'
+import TextField from '../../../../../components/ui/text-field'
+import { CONNECT_HARDWARE_ROUTE } from '../../../../../helpers/constants/routes'
+const ENTER_KEY = 13
+class EmailScreen extends PureComponent {
+  constructor (props) {
+    super(props)
+    this.state = {
+      email: null,
+    }
+    this.history = this.props.history
```

Comment:
This class field seems unnecessary, esp. since it's used once? Can we inline this?